src/Page.js

```diff
@@ -87,6 +87,9 @@ function Page(pageConfig) {
   this.keywords = {};
   this.navigableHeadings = {};
   this.pageSectionsHtml = {};
+
+  // flag to indicate whether this page has a site nav
```

Comment:
A quick search shows majority of inline comments have capitalised starting character (although not all).